runtime/src/tokio/runtime.rs

```diff
@@ -371,6 +371,13 @@ pub struct Context {
     children: Arc<Mutex<Vec<AbortHandle>>>,
 }
 
+impl Context {
+    /// Access the [Metrics] of the runtime.
+    fn metrics(&self) -> &Metrics {
```

Comment:
The `macro` allows us to use a reference here (rather than cloning the arc). A small win for production env efficiency 🚀 .